### PR TITLE
fix(): update json-formatter.component.ts

### PR DESCRIPTION
**Pull request by christrude**
## Description
Helping the child parser not do double the work.

Part of an investigation to the formatter not being able to handle large files

### What's included?
- Improved parseChildren function

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

File: src/platform/core/json-formatter/json-formatter.component.ts [hunk 1]
```diff
@@ -212,10 +212,7 @@ export class TdJsonFormatterComponent {
 
   private parseChildren(): void {
     if (this.isObject()) {
-      this._children = [];
-      for (const key of Object.keys(this._data)) {
-        this._children.push(key);
-      }
+      this._children = Object.keys(this._data) ? Object.keys(this._data) : [];
```
Comment (christianmemije) [inline src/platform/core/json-formatter/json-formatter.component.ts:215]: Can't we just do `this._children = Object.keys(this._data);`? An empty object would just return an empty array.

**Comment by christrude** [comment]
> Commit messages are invalid format. I would recommend squashing to fix the first commit message. Also follow https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines

I thought i amended the commit, thats why i was confused that it still errored. I'll squash instead.

File: src/platform/core/json-formatter/json-formatter.component.ts (hunk 1 above)
Comment (christrude) [reply src/platform/core/json-formatter/json-formatter.component.ts:215]: Yeah just double checked. Can push that.

**Comment by christianmemije** [comment]
@christrude Almost there, just need the commit message format `fix(json-formatter): optimize`

**Pull request merged**